org.eclipse.acute/src/org/eclipse/acute/dotnetrun/DotnetRunTab.java

```diff
@@ -103,6 +104,8 @@ public void setDefaults(ILaunchConfigurationWorkingCopy configuration) {
 		configuration.setAttribute("PROJECT_ARGUMENTS", "");
 		configuration.setAttribute("PROJECT_BUILD", true);
 		configuration.setAttribute("PROJECT_RESTORE", true);
+		configuration.setAttribute(DebugPlugin.ATTR_PATH, "dotnet");
+		configuration.setAttribute(DebugPlugin.ATTR_WORKING_DIRECTORY, pathText.getTabs());
```

Comment:
setDefaults is called before createControl, so this line gets a NPE on pathText. Also, did you mean getText() instead of getTabs()?